pavelib/assets.py

```diff
@@ -384,14 +384,24 @@ def coffeescript_files():
 
 @task
 @no_help
-def compile_coffeescript(*files):
+def compile_coffeescript(*coffee_files):
     """
     Compile CoffeeScript to JavaScript.
     """
-    if not files:
-        files = ["`{}`".format(coffeescript_files())]
+    js_files = ""
+
+    if not coffee_files:
+        coffee_files = ["`{}`".format(coffeescript_files())]
+        js_files = r"`{} | sed 's/\.coffee/\.js/g'`".format(coffeescript_files())
+    else:
+        js_files = r"`{} | sed 's/\.coffee/\.js/g'`".format(*coffee_files)
```

Comment:
Why are we shelling out for this, rather than just converting `.coffee` to `.js` in python?

    if not coffee_files:
        coffee_files = coffeescript_files()
    
    js_files = [re.sub('coffee$', 'js', file) for file in coffee_files]